src/fluvio.rs

```diff
@@ -103,10 +132,21 @@ impl Fluvio {
         let rc = self.inner.clone();
 
         let promise = future_to_promise(async move {
-            rc.topic_producer(topic)
+            info!("Producing topic: {:#?}", &topic);
+
+            let result = rc
+                .topic_producer(&topic)
                 .await
                 .map(|producer| JsValue::from(TopicProducer::from(producer)))
                 .map_err(|e| (FluvioError::from(e).into()))
+                .and_then(|r| {
+                    info!("Produced topic: {:#?}", &topic);
```

Comment:
Maybe a more appropriate log message would be something like: "Created producer for topic {}", topic"